scripts/release/check_sync_criteria.py

```diff
@@ -96,20 +96,28 @@ def check_sync_criteria(
         for pkg_name in sorted(build_file.sync_packages):
             print('-', pkg_name)
 
-    # check that count is satisfied
-    if build_file.sync_package_count is not None:
+    # check that count is satisfied; if it exists, sync_package_percent is
+    # preferred over sync_package_count
+    packages_needed = None
```

Comment:
Can you choose a different name for this value? `packages_needed` makes it sound like a collection of packages that are needed in order for the sync to occur, which would be less confusing if not for the fact that that's actually a valid configuration option. Maybe `minimum_packages` or `min_package_count` or `package_count_needed`?

Reply:
Yeah, good point.  I've changed it to `sync_package_count` in 5fb4a93.